flexmeasures/data/models/reporting/pandas_reporter.py

```diff
@@ -0,0 +1,114 @@
+import pandas as pd
+from flexmeasures.data.models.reporting import Reporter
+from flexmeasures.data.schemas.reporting.pandas_reporter import (
+    PandasReporterConfigSchema,
+)
+
+
+class PandasReporter(Reporter):
+    """This reporter applies a series of pandas methods on"""
+
+    __version__ = "1"
+    __author__ = None
+    schema = PandasReporterConfigSchema()
+
+    def deserialize_report_config(self):
+        # call super class deserialize_report_config
+        super().deserialize_report_config()
+
+        # extract PandasReporter specific fields
+        self.transformations = self.reporter_config.get("transformations")
+        self.final_df_output = self.reporter_config.get("final_df_output")
+
+    def _compute(self) -> pd.Series:
+        """"""
+        # apply pandas transformations to the dataframes in `self.data``
+        self.apply_transformations()
+
+        final_output = self.data[self.final_df_output]
+        return final_output
+
+    def get_object_or_literal(self, value, method):
+        """This method allows using the dataframes as inputs of the Pandas methods that
+        are run in the transformations. Make sure that they have been created before accessed.
+
+        This works by putting the symbol `@` in front of the name of the dataframe that we want to reference.
+        For instance, to reference the dataframe test_df, which lives in self.data, we would do `@test_df`.
+
+        This functionality is disabled for methods `eval`and `query` to avoid interfering their internal behaviour
+        given that they also use `@` to allow using local variables.
+
+        Examples
+        >> self.get_object_or_literal(["@sensor_1", "@sensor_2"], "sum")
+        [[                    ...n: 0:00:00,                     ...n: 0:00:00]]
```

Comment:
Those are the `__repr__` of the `BeliefDataFrame`/`pandas.DataFrame` objects that are reference with `"@sensor_1"` and   `"@sensor_2"` in this list. I've change this so that it's more clear. Let me know if we should use a different object representation or, maybe, to add a description.

Reply:
Thanks. I like the shorthand representation of the resulting BeliefsDataFrames.

But it still confuses me, because:
- the chosen variable names suggest that this function is taking a description of two variables containing sensor objects (with ID = 1 and 2), and fetching their data in the form of a BeliefsDataFrame. Better use variable names that suggest the correct type (and in practice, I wouldn't expect anyone to use sensor IDs as part of the variable names), e.g. `["@df_wind", "@df_solar"]`.
- Summing VAT to PV data seems odd. Better add data from 2 power sensors instead (either both production, or both consumption). I realize this function doesn't actually do the summation, but nevertheless.
- Printing the result would show single brackets `[` rather than double brackets `[[`.

Reply:
Upon testing, it looks like the `sensor_1` syntax did have a special meaning. I'm submitting a PR to your branch that implements your "alias" idea, so the df_wind and df_solar example then make more sense again.

Reply:
Great, thanks for the contributions!